bin/riak-bucket-exporter.js

```diff
@@ -113,17 +123,28 @@ function processKey(key, cb) {
     var out = {key: key};
     out.indexes = extractIndexes(meta);
     out.data = obj;
+    if(program.meta){
```

Comment:
my personal favorite formatting would be `if (program.meta) {` with two spaces included. but I guess thats not really important for such a small thing